src/architecture/transactions.md

```diff
@@ -42,7 +42,7 @@ various constraints based on this key.
     the [light client library](https://github.com/exonum/exonum-client).
 
 !!! note "Example"
-    In a sample [cryptocurrency service][cryptocurrency],
+    In the [How to Create a Service][demo-service] tutorial,
```

Comment:
You call it tutorial but you give a link for the demo code. Shall we say 
> In the [sample service](demo-service) demo...

??